pom.xml

```diff
@@ -157,6 +157,11 @@ THE SOFTWARE.
             <artifactId>structs</artifactId>
             <version>1.20</version>
         </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm</artifactId>
+            <version>9.1</version>
```

Comment:
I think this may conflict with older versions and the like.
would it be better to bump the jenkins core version so that all its transitive dependencies are met?

Reply:
In fact this is already bundled in the plugin.
Unclear why this would fail in the PCT but it is also using maskClasses. The code seems to run fine in a Jenkins 2.303 version.

I would suggest this is closed as a Parent CT false positive

Reply:
+1